tests/codeception/common/_support/FixtureHelper.php

```diff
@@ -58,6 +58,30 @@ public function globalFixtures()
     }
 
     /**
+     * Workaround used to load custom fixtures for the test
+     * @link https://github.com/Codeception/Codeception/issues/2258
+     * @link https://github.com/yiisoft/yii2/issues/5575
+     * @param array $fixtureConfig
+     */
+    public function loadFixturesForTest(array $fixtureConfig)
+    {
+        $fixtures = ArrayHelper::merge(
```

Comment:
it is not ugly since this is done in separated meaningfull method name. As was said, for your case you need to load them all, for the application case you need only to unload `User` fixture instead of calling `User::deleteAll()`, can you fix it? So to summarize:

- `getFixture` and `loadFixtures($fixtures)` are public. 
- `User::deleteAll()` replaced with `unloadFixtures`